src/test/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/javadocparagraph/InputJavadocParagraphCheck1.java

```diff
@@ -31,4 +32,39 @@ class Check {
     */
    // violation 2 lines above '<p> tag should be preceded with an empty line.'
     void inheritDocWithThrows() {}
+
+    /**
+     * Some summary.
+     *
+     * <p>The following package declaration:</p>
+     * <pre>
+     * package com.puppycrawl.tools.checkstyle;
+     * </pre>
+     */
+    int iamtoolazyyyyyyy = 45;
+
+    /**
+     * Some summary.
+     *
+     *<p>
+     *  <ul>
```

Comment:
This should be violation, why it is not ?
Even we ignore \n , there are bunch of spaces before UL.
And should be no P before UL, as it is blocktag

Reply:
> Even we ignore \n , there are bunch of spaces before UL.

we decided not to check this at https://github.com/checkstyle/checkstyle/pull/15686#discussion_r1788518551, read my comment above it for explanation

> And should be no P before UL, as it is blocktag

fixed this, thanks for pointing this out

Reply:
please create issue and put "ok , until #xxxxxxxxx" to let readers know that we detected it but postponed this false-negative

Reply:
Please put new issue in several case of this PR to make it clear it is postponed

Reply:
this will be addressed at https://github.com/checkstyle/checkstyle/issues/15762

Reply:
I have added `// ok until 15762` comments